### PR TITLE
Add 2200 Support to Server Hardware and Interconnect 

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
Connection Template
Interconnect
Interconnect types
Server Hardware
Server Hardware Types

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by nabhajit-ray — APPROVED** [review]

File: endpoints-support.md [hunk 1]
```diff
@@ -87,23 +87,23 @@
...
 |     **Interconnect Link Topologies**                                                                                                             |
-|<sub>/rest/interconnect-link-topologies</sub>                                            | GET      | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
-|<sub>/rest/interconnect-link-topologies/{id}</sub>                                       | GET      | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/interconnect-link-topologies</sub>                                            | GET      | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:90]: are you supporting these endpoints..?

File: endpoints-support.md [hunk 2]
```diff
@@ -87,23 +87,23 @@
...
+|<sub>/rest/interconnects</sub>                                                           | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/interconnects/{id}</sub>                                                      | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/interconnects/{id}</sub>                                                      | PATCH    | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md]: added twice.. please check

Comment (yuvirani) [reply endpoints-support.md]: Done

File: endpoints-support.md (hunk 1 above)
Comment (yuvirani) [reply endpoints-support.md:90]: yes

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**